### DISCUSSION TITLE
Modifying the View of :Git

### DISCUSSION THREAD

**Discussion by DyBev** [tpope/vim-fugitive#2163] [Q&A]
I would like to make :Git be able to open in a floating window, I find the opening a split rather intrusive for the work flow, I'm currently using 'G to jump to the :Git window without opening a new buffer, would be nice to know if I could directly change the config

Any way to do that?

**Comment by tpope** [comment]
There's no built-in support for this. `:Git` does little more than `:split fugitive://...`, so you could load that URL directly into a floating window if you wanted to. You can retrieve the URL with `FugitiveFind(':')`.
